src/components/block/api.ts

```diff
@@ -0,0 +1,152 @@
+import Block from './index';
+import {BlockToolData, ToolConfig} from '../../../types/tools';
+import {SavedData} from '../../types-internal/block-data';
+import {BlockAPI as BlockAPIInterface} from '../../../types/api';
+
+function BlockAPI(block: Block) {
+  const blockAPI: BlockAPIInterface =  {
+    /**
+     * Tool name
+     *
+     * @return {string}
+     */
+    get name(): string {
+      return block.name;
+    },
+
+    /**
+     * Tool config passed on Editor's initialization
+     *
+     * @return {ToolConfig}
+     */
+    get settings(): ToolConfig {
+      return block.settings;
+    },
+
+    /**
+     * .ce-block element, that wraps plugin contents
+     *
+     * @return {HTMLElement}
+     */
+    get holder(): HTMLElement {
+      return block.holder;
+    },
+
+    /**
+     * Tool contents holder, returned from Tool's render() method
+     *
+     * @return {HTMLElement}
+     */
+    get pluginsContent(): HTMLElement {
```

Comment:
@neSpecc @khaydarov I think we need to rename this as `plugin` is never used for any Editor.js entity